deepmd/utils/data_system.py

```
@@ -241,7 +241,7 @@ def _load_test(self, ntests=-1):
                 self.test_data[nn].append(test_system_data[nn])
 
     @property
-    @lru_cache(maxsize=None)
+    @cache
```

Ruff

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)